packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -1323,3 +1323,67 @@ export async function t9n(componentTestSetup: ComponentTestSetup): Promise<void>
     await page.waitForChanges();
   }
 }
+
+/*
+ * @param {ComponentTestSetup} ComponentTestSetup - A component tag, html, or the tag and e2e page for setting up a test.
+ * @param {string} toggleProp - Toggle property to test. Currently, either "open" or "expanded".
+ * @param {string} containerSelector - Container element selector to test for visibility.
```

Comment:
Testing for visibility seems out of scope for this test helper. `onToggleOpenCloseComponent` helps components easily emit respective open/close events, but doesn't handle visibility, right?

Reply:
Right. We're passing the selector in to `expect(await container.isVisible()).toBe(true); ` before we await for the events. I rewrote it based on some previous comments to pass this instead.     
`emitsOpenCloseEvents(simpleDropdownHTML, "open", `${CSS.calciteDropdownGroup} >>> .${CSS.container}`);`
Does that look ok? I can rewrite that bit to say `Container element selector.`